### ISSUE TITLE
Minor dependency missing

### ISSUE THREAD

**Issue by MountainLogic** [ShaneFlandermeyer/gr-plasma#16]
I needed to add **qtbase5-dev** before I could build.  Also obviously **gnuradio-dev** and **UHD** are needed.

**Comment by MountainLogic** [comment]
Also found on a fresh Ubuntu 24.04 install that has python 3.12 as default python install, that I also needed to apt install python 3.10 AND python3.10-dev from dead snake ppa

**Comment by Mat-DB** [comment]
@MountainLogic 
> I needed to add qtbase5-dev before I could build.

How did you solve the problem?

Edit: Never mind, I tried doing `sudo apt install qtbase5-dev` and that solved it.

**Comment by MountainLogic** [comment]
Yes, `sudo apt install qtbase5-dev` fixes the dependency.  I entered this as a minor issue as it should be added to the Install readme.  I was just a bit to lazy to make a pull request with the readme fix.   The install instructions should be changed from:

`sudo apt install nlohmann-json3-dev libqwt-qt5-dev `

to 

`sudo apt install nlohmann-json3-dev libqwt-qt5-dev qtbase5-dev`
